src/source/reference/get-metrics-ads.rst

```diff
@@ -61,7 +61,7 @@ L3 Category           ``L3Category``       Level 3 category name, if applicable
 Title                 ``title``            Title of the ad
 Start Date            ``startDate``        Creation date of the ad
 End Date              ``endDate``          If the ad is deleted, deletion date of the ad
-CPC                   ``cpc``              CPC of the ad for which performance metrics are calculated, in Euros
```

Comment:
we should change "Euros" to "Local Currency"

Reply:
erm, yes good catch, and I see we need to fix some prod properties :) 
`kjca.fr_CA.metric.report.column.spent=Dépense totale (EUR)`